packages/synapse-react-client/src/components/trash/TrashCanList.tsx

```diff
@@ -38,6 +38,7 @@ function TrashCanListItem(props: TrashCanListItemProps) {
       </td>
       <td>{item.entityId}</td>
       <td>{item.entityName}</td>
+      <td>{item.entityType}</td>
```

Comment:
Use `entityTypeToFriendlyName` from EntityTypeUtils.ts

Reply:
Thanks Nick! i just updated that.